tabular/src/autogluon/tabular/models/tabular_nn/hyperparameters/parameters.py

```diff
@@ -76,6 +76,62 @@ def get_hyper_params():
     return hyper_params
 
 
+def get_quantile_fixed_params():
+    """ Parameters that currently cannot be searched during HPO """
+    params = {
+        'num_updates': 20000,  # maximum number of updates
+        'updates_wo_improve': 500,  # we terminate training if validation performance hasn't improved in the last 'updates_wo_improve' # of updates
+        'seed_value': None,  # random seed for reproducibility (set = None to ignore)
+        # For data processing:
+        'proc.embed_min_categories': 4,  # apply embedding layer to categorical features with at least this many levels. Features with fewer levels are one-hot encoded. Choose big value to avoid use of Embedding layers
+        # Options: [3, 4, 10, 100, 1000]
+        'proc.impute_strategy': 'median',  # strategy argument of sklearn.SimpleImputer() used to impute missing numeric values
+        # Options: ['median', 'mean', 'most_frequent']
+        'proc.max_category_levels': 100, # maximum number of allowed levels per categorical feature
+        # Options: [10, 100, 200, 300, 400, 500, 1000, 10000]
+        'proc.skew_threshold': 0.99,  # numerical features whose absolute skewness is greater than this receive special power-transform preprocessing. Choose big value to avoid using power-transforms
+        # Options: [0.2, 0.3, 0.5, 0.8, 1.0, 10.0, 100.0]
+
+        'max_batch_size': 512,  # max batch-size
+    }
+    return params
+
+
+def get_quantile_hyper_params():
+    """ Parameters that currently cannot be searched during HPO """
```

Comment:
these can be tuned during HPO right?

Reply:
Rather than copy/paste, it would be preferrable if you just called `dict = get_hyper_params()` inside this function and then modified the `dict` appropriately from there.

Reply:
Yes, it works with HPO.
Modified to reuse the existing hyperparams.

Reply:
since these hyperparams can be searched in HPO, the docstring is wrong:

""" Parameters that currently cannot be searched during HPO """

Reply:
Yes, I just modified it.